packages/formstr-app/src/containers/CreateFormNew/components/AnswerSettings/Conditions/LogicSelector.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { Select, Typography } from "antd";
+import { ConditionRule, ConditionGroup, isConditionRule } from "./types";
+
+const { Text } = Typography;
+
+interface LogicSelectorProps {
+  item: ConditionRule | ConditionGroup;
+  isLastItem: boolean;
+  onLogicChange: (value: "AND" | "OR") => void;
+  isGroupLogic?: boolean; 
+}
+
+const LogicSelector: React.FC<LogicSelectorProps> = ({
+  item,
+  isLastItem,
+  onLogicChange,
+  isGroupLogic = false,
+}) => {
+  if (isLastItem) return null;
+
+  const logicValue = item.nextLogic || "AND";
+
+  const connectionText = isGroupLogic 
+    ? "Connect with next rule set:" 
+    : "Connect with next condition:";
+
+  return (
+    <div className="logic-selector">
+      <div style={{ display: 'flex', alignItems: 'center', marginBottom: '8px' }}>
+        <div style={{ 
+          width: '24px', 
+          height: '24px', 
+          display: 'flex', 
+          alignItems: 'center', 
+          justifyContent: 'center', 
+          background: logicValue === 'AND' ? '#1890ff' : '#ff4d4f',
```

Comment:
let's not use magic values, and move colors to a const atleast if not a themed object